ui/widgets/tooltip.js

```diff
@@ -265,6 +266,9 @@ $.widget( "ui.tooltip", {
 		tooltipData = this._find( target );
 		if ( tooltipData ) {
 			tooltipData.tooltip.find( ".ui-tooltip-content" ).html( content );
+			if ( this.options.textDir ) {
+				this._applyTextDir( tooltipData.tooltip.find( ".ui-tooltip-content" ) );
```

Comment:
By no means. In this case the tooltip.find returns JQuery object and '_applyTextDir' sets its 'direction' style